src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
According to the checklist, remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you can modify this line to: `font-family: Roboto, Arial, Helvetica, sans-serif, sans-serif;`